docs/01-basics/02-timing-didactics-and-more.adoc

```diff
@@ -3,37 +3,44 @@
 include::../config/_feedback.adoc[]
 // end::FEEDBACK[]
 
+// tag::REMARK[]
+[IMPORTANT]
+====
+Bitte die Anzahl der Tage sowie der erreichbaren Punkte oben bei den **?** einsetzen.
+====
+// end::REMARK[]
+:recommended-duration: **?**
```

Comment:
I’d prefer `recommended-duration-in-days` because it is easier to understand, more explicit.